daemon/cmd/daemon_main.go

```diff
@@ -1090,17 +1106,6 @@ func initEnv() {
 		neighborsmap.SizeofNeighKey6+neighborsmap.SizeOfNeighValue,
 		lbmap.SizeofSockRevNat6Key+lbmap.SizeofSockRevNat6Value)
 
-	// Prepopulate option.Config with options from CLI.
-	option.Config.Populate(Vp)
```

Comment:
A bunch of other stuff in `initEnv` relies on option.Config -- is this going to be a problem?

Reply:
Yep, the order remains the same, I basically just split it so that all commands get `option.Config.Populate` but only root command calls `initEnv`.